arches_lingo/src/arches_lingo/components/sidenav/components/NavReferenceData.vue

```diff
@@ -0,0 +1,47 @@
+<script setup lang="ts">
+import { ref } from "vue";
+import { useGettext } from "vue3-gettext";
+
+import { routeNames } from "@/arches_lingo/routes.ts";
+import SideNavSection from "@/arches_lingo/components/sidenav/SideNavSection.vue";
+
+import type { SideNavMenuItem } from "@/arches_lingo/types.ts";
+
+const { $gettext } = useGettext();
+
+const props = defineProps<{
+    item: SideNavMenuItem;
+}>();
+
+const children = <SideNavMenuItem[]>[
+    {
+        key: "contributors",
+        label: $gettext("Contributors"),
+        icon: "fa fa-user",
```

Comment:
We should use font awesome in vue components. Could you replace these with the closest `pi` equivalent?